plugins/chat/assets/javascripts/discourse/components/chat/message-creator/lib/chatables-loader.js

```diff
@@ -74,24 +81,32 @@ export default class ChatablesLoader {
         let chatable;
         if (channel.chatable?.users?.length === 1) {
           chatable = ChatChatable.createUser(channel.chatable.users[0]);
-          chatable.tracking = this.#injectTracking(chatable);
         } else {
           chatable = ChatChatable.createChannel(channel);
-          chatable.tracking = channel.tracking;
         }
+
+        chatable.tracking = channel.tracking;
+        chatable.unread_thread_count =
+          channel.unreadThreadsCountSinceLastViewed;
         return chatable;
       })
       .filter(Boolean)
       .slice(0, MAX_RESULTS);
   }
 
-  #injectTracking(chatable) {
-    if (!chatable.type === "channel") {
+  #findChannel(chatable) {
+    if (!["user", "channel"].includes(chatable.type)) {
       return;
     }
 
-    return this.chatChannelsManager.allChannels.find(
-      (channel) => channel.id === chatable.model.id
-    )?.tracking;
+    const { allChannels } = this.chatChannelsManager;
+    if (chatable.type === "user") {
+      return allChannels.find(
+        ({ chatable: { users } }) =>
+          users?.length === 1 && users[0].id === chatable.model.id
+      );
```

Comment:
Since chatable type for 1-1 DMs is `user` when searching within message creator, we need to get the channel so that we can access tracking data.